pages/docs/data-ops/continuous-integration.mdx

```diff
@@ -4,9 +4,13 @@ description: Hashboard can serve to run continuous integration on your Business
 
 ## Continuous integration
 
-Hashboard can serve to run continuous integration on your Business Intelligence dashboards, this can be integrated with git and dbt workflows in Hashboard. You can invoke the Hashboard CLI in a continuous integration system to automatically generate previews or deploy your project.
+By combining Hashboard's CLI and preview features, you can integrate your entire BI stack into a continuous integration workflow.
```

Comment:
Maybe something like: "you can even preview data changes in data pipelines and see how they will affect your dashboards and reporting"